src/app-gocardless/banks/berliner_sparkasse_beladebexxx.js

```diff
@@ -0,0 +1,98 @@
+import {
+  printIban,
+  amountToInteger,
+  sortByBookingDateOrValueDate,
+} from '../utils.js';
+
+/** @type {import('./bank.interface.js').IBank} */
+export default {
+  institutionIds: ['BERLINER_SPARKASSE_BELADEBEXXX'],
+
+  accessValidForDays: 90,
+
+  normalizeAccount(account) {
+    return {
+      account_id: account.id,
+      institution: account.institution,
+      mask: account.iban.slice(-4),
+      iban: account.iban,
+      name: [account.name, printIban(account)].join(' '),
+      official_name: account.product,
+      type: 'checking',
+    };
+  },
+
+  /**
+   * Following the GoCardless documentation[0] we should prefer `bookingDate`
+   * here, though some of their bank integrations uses the date field
+   * differently from what's describen in their documentation and so it's
+   * sometimes necessary to use `valueDate` instead.
+   *
+   *   [0]: https://nordigen.zendesk.com/hc/en-gb/articles/7899367372829-valueDate-and-bookingDate-for-transactions
+   */
+  normalizeTransaction(transaction, _booked) {
+    const date =
+      transaction.bookingDate ||
+      transaction.bookingDateTime ||
+      transaction.valueDate ||
+      transaction.valueDateTime;
+
+    // If we couldn't find a valid date field we filter out this transaction
+    // and hope that we will import it again once the bank has processed the
+    // transaction further.
+    if (!date) {
+      return null;
+    }
+
+    let remittanceInformationUnstructured;
+
+    if (transaction.remittanceInformationUnstructured) {
+      remittanceInformationUnstructured =
+        transaction.remittanceInformationUnstructured;
+    } else if (transaction.remittanceInformationStructured) {
+      remittanceInformationUnstructured =
+        transaction.remittanceInformationStructured;
+    } else if (transaction.remittanceInformationStructuredArray?.length > 0) {
+      remittanceInformationUnstructured =
+        transaction.remittanceInformationStructuredArray?.join(' ');
+    }
+
+    if (transaction.additionalInformation)
+      remittanceInformationUnstructured +=
+        ' ' + transaction.additionalInformation;
+
+    const usefulCreditorName =
+      transaction.ultimateCreditor ||
+      transaction.creditorName ||
+      transaction.debtorName;
+
+    return {
+      ...transaction,
+      creditorName: usefulCreditorName,
+      remittanceInformationUnstructured: remittanceInformationUnstructured,
+      date: transaction.bookingDate || transaction.valueDate,
+    };
+  },
+
+  sortTransactions(transactions = []) {
+    return sortByBookingDateOrValueDate(transactions);
```

Comment:
This seems like the default case.  Could you import and use the fallback version instead?

See this file for an example: https://github.com/actualbudget/actual-server/blob/master/src/app-gocardless/banks/belfius_gkccbebb.js